### ISSUE TITLE
Cannot connecto to camera through ip/http

### ISSUE THREAD

**Issue by webberd1wei** [fritz-smh/yi-hack#206]
I am new to this repo. I configured the sd card and put it in Yi home camera. The LED became solid blue. But when I tried to connect to the camera through http://mycameraip/ on my browser, I cannot access it. My firmware is 1.8.7.0D_201708091510. I read a couple of open issues regarding firmware updates, does this hack work with my version of firmware? If not directly, are there workarounds?

**Comment by Clem-** [comment]
Hi,
When you say `mycameraip`, is that the *local* ip address?

**Comment by webberd1wei** [comment]
@Clem- yep, 192.168.1.X.

**Comment by Clem-** [comment]
Are your sure that your router's assigning this very ip address to your camera?

**Comment by webberd1wei** [comment]
Yes I am sure. I assigned static ip to it. Does this hack work with the version of firmware I have?

**Comment by Clem-** [comment]
Im personally on 1.8.5.1M_201512011815
Can’t say much about your version. 
Did you find errors in your log.txt (in `test` dir)?

**Comment by webberd1wei** [comment]
@Clem- Thanks for your patience. I did not find log.txt in `test` dir. Is that maybe the problem?

**Comment by Clem-** [comment]
That could explain one thing : the hack didn’t boot because of your firmware version. 

If you really want to benefit this hack, you should downgrade to 1.8.5
